week0001/phandinhthe/Week0001.java

```diff
@@ -0,0 +1,23 @@
+package phandinhthe;
+public class Week0001 {
+    public static void main (String[] args) {
+      int additionFor3 = 0;
+      int additionFor5 = 0;
+      int selectedAddition = 1;
+
+      for(int i=1; i<101;) {
+        if (i%3 == 0)
+          System.out.println("Fizz");
+        if (i%5 == 0)
+          System.out.println("Buzz");
+          
+        additionFor3 = 3-(i%3);
+        additionFor5 = 5-(i%5);
+
+        selectedAddition = additionFor3 > additionFor5 ?
```

Comment:
Cool, this looks like a smart solution that jumps to the nearest multiple of 3 or 5.